java/openjdk/Portfile

```diff
@@ -101,7 +104,7 @@ subport openjdk7-zulu {
 }
 
 subport openjdk8 {
-    version      8u302
+    version      8u322
```

Comment:
I don't see why this version needs to be bumped. I get that it's tempting to let it match the latest OpenJDK version, but bumping this version will just cause an unnecessary update for everyone who has this empty port installed without any functional benefit.